pkg/models/observer_buffer.go

```diff
@@ -55,6 +61,33 @@ type ObserverBuffer struct {
 	MaxE2ELatency       time.Duration
 	MinE2ELatency       time.Duration
 	SumE2ELatency       time.Duration
+
+	InvalidErrors map[MetadataCodeDescription]int
+	FailedErrors  map[MetadataCodeDescription]int
```

Comment:
Does 'failed' always contain retryable failures? If so I wonder if we can take a step towards rectifying poor naming choices of the past?